homeassistant/components/nmbs/sensor.py

```diff
@@ -114,11 +122,14 @@ def unique_id(self):
 
     @property
     def icon(self):
-        """Return the default icon or an alert icon if delays."""
-        if self._attrs and int(self._attrs["delay"]) > 0:
-            return DEFAULT_ICON_ALERT
+        """Return the default icon or an alert icon if delays or cancellations."""
+        if self._attrs:
+            delay = seconds_to_minutes(self._attrs["delay"])
+            departure_canceled = bool(int(self._attrs["canceled"]))
+            if delay > 0 or departure_canceled:
+                return "mdi:alert-octagon"
 
-        return DEFAULT_ICON
+        return "mdi:train"
```

Comment:
What's wrong with the [`DEFAULT_ICON`](https://github.com/home-assistant/core/blob/80a7f86f47b37dd41e217c68259b8b0f2466abbd/homeassistant/components/nmbs/sensor.py#L25)?